benchmarks/BayesianInference/Project.toml

```diff
@@ -15,16 +15,16 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Turing = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
 
 [compat]
-BenchmarkTools = "1.0"
-DiffEqBayes = "3.0.1"
-DiffEqCallbacks = "2"
-Distributions = "0.25"
-DynamicHMC = "3.1"
-OrdinaryDiffEq = "6"
-ParameterizedFunctions = "5.3"
-Plots = "1.4"
-RecursiveArrayTools = "2.5"
-SciMLBenchmarks = "0.1"
-StanSample = "7"
-StaticArrays = "1"
-Turing = "0.28, 0.30"
+BenchmarkTools = "1.0, 1.6.0"
+DiffEqBayes = "3.2.1, 3.6.2"
+DiffEqCallbacks = "2, 4.0.0"
+Distributions = "0.25, 0.25.112"
+DynamicHMC = "3.1, 3.4.7"
+OrdinaryDiffEq = "6, 6.89.0"
+ParameterizedFunctions = "5.3, 5.17.0"
+Plots = "1.4, 2.0.0"
+RecursiveArrayTools = "2.5, 3.27.1"
+SciMLBenchmarks = "0.1, 0.1.3"
+StanSample = "7, 7.10.1"
+StaticArrays = "1, 1.9.8"
+Turing = "0.28, 0.30, 0.34.1, 0.35.1"
```

Comment:
Remove the lower ones and bump it?

Reply:
@ChrisRackauckas I removed the lower versions and tried to bump them but I am getting an unsatisfiable requirements error for almost every module.

Reply:
> Remove the lower ones and bump it?

 I removed the lower versions and tried to bump them but I am getting an unsatisfiable requirements error for almost every module after removing older versions

Reply:
Update to https://github.com/JuliaRegistries/General/pull/118430

Reply:
@ChrisRackauckas Done